README.md

```diff
@@ -64,6 +66,21 @@ if result.status_code == 200:
 print(result.text)
 ```
 
+#### With [HTTPX](https://www.python-httpx.org/) library
+
+```python
+import httpx
+from mauth_client.httpx_mauth import MAuthHttpx
+
+# MAuth configuration
+APP_UUID = "<MAUTH_APP_UUID>"
+private_key = open("private.key", "r").read()
```

Comment:
I know this is a copy of the requests mauth example but usually we take our secrets directly from env these days so i wonder if it's more fitting to show that in the example